kotlin-mbedtls/src/main/kotlin/org/opencoap/ssl/transport/DtlsServer.kt

```diff
@@ -132,9 +132,15 @@ class DtlsServer(
 
     fun numberOfSessions(): Int = executor.supply { sessions.size }.join()
 
-    fun setSessionAuthenticationContext(adr: InetSocketAddress, authenticationContext: String): CompletableFuture<Boolean> =
+    fun putSessionAuthenticationContext(adr: InetSocketAddress, key: String, creds: String?): CompletableFuture<Boolean> =
```

Comment:
Could you still rename parameter `creads` into something more like generic, like `value` or `contextValue`?